easybuild/easyblocks/a/amber.py

```diff
@@ -315,8 +323,8 @@ def install_step(self):
 
         super(EB_Amber, self).install_step()
 
-        # Run the tests located in the build directory
-        if self.cfg['runtest']:
+        # Run the tests located in the build directory, only for Amber. Tests fail when building only AmberTools
+        if self.cfg['runtest'] and self.name == 'Amber':
```

Comment:
do all tests fail with AmberTools, or only some?
if not all tests fail, I think it's still useful to run them in combination with EB option `--ignore-test-failure`.

Reply:
I did not try to untangle the tests, but not all executables from Amber get created, and the tests assume that they are there.

Reply:
See #2604 (and https://github.com/easybuilders/easybuild-easyconfigs/pull/14028) - it is trying to run the Amber test suite and not the AmberTools test suite. There were still some failures with the AmberTool test suite and I've not had the chance to investigate further.

Reply:
Yeah, we're doing the runtest part wrong. I think I have the correct stuff in my test-.eb but I got sidetracked and haven't had time to go back and revisit.
It should not matter if we build Amber or AmberTools the test part (CMake and old style) is written so that it only tests what is actually built, if we call it correctly. I checked with the devs on that.

Reply:
@akesandgren  did you hear back from the Amber devs yet?

Reply:
They basically said what I said above, The CMake part should do the right thing when asked to run make test.
But I haven't had time to revisit this on my side...

Reply:
This still fails as of today. Can't build AmberTools without disabling tests.